File: googlecalendar/Front End/movieNight/src/app/models/Movie.ts
export  class Movie{
    imdbID:string;
     Title:string;
     imdbRating:string;
     year:string;
     Poster:string;
      constructor(){}
      //**** Setters */
      set imdbId(_imdbID:string) {
        this.imdbID=_imdbID;
     }
     set title(_Title:string) {
        this.Title=_Title;
     }
     set imdBRating(_imdbRating:string) {
        this.imdbRating=_imdbRating;
     }
     set releaseYear(_year:string) {
        this.year=_year;
     }
     set poster(_poster:string) {
        this.Poster=_poster;
     }
     /////######## getters #############
  get imdbId(): string {
    return this.imdbID;
  }
  get title(): string {
    return this.Title;
  }
  get imdBRating(): string {
    return this.imdbRating;
  }
  get releaseYear(): string {
    return this.year;
  }
  get poster(): string {
    return this.Poster;
  }












}

File: googlecalendar/src/main/java/com/movie/googlecalendar/model/Movie.java
package com.movie.googlecalendar.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.io.Serializable;



@Document(collection = "movie")

@JsonIgnoreProperties(ignoreUnknown = true)

public class Movie  {



    @Id
    private String imdbID;

    @JsonProperty("Title")

    private String Title;

    private String imdbRating;

    @JsonProperty("Year")

    private String Year;

    @JsonProperty("Poster")

    private String Poster;



    public Movie() {

    }



    public String getImdbID() {

        return imdbID;

    }



    public void setImdbID(String imdbID) {

        this.imdbID = imdbID;

    }



    public String getTitle() {

        return Title;

    }



    public void setTitle(String title) {

        title = title;

    }



    public String getImdbRating() {

        return imdbRating;

    }



    public void setImdbRating(String imdbRating) {

        this.imdbRating = imdbRating;

    }



    public String getYear() {

        return Year;

    }



    public void setYear(String year) {

        year = year;

    }



    public String getPoster() {

        return Poster;

    }



    public void setPoster(String poster) {

        poster = poster;

    }

}